src/pages/ProductList/ProductList.js

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import Product from "../../components/Product";
+
+export default function ProductList() {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    fetch("/products")
+      .then((res) => res.json())
+      .then((data) => {
+        setProducts(data);
+      });
+  }, []);
```

Comment:
이렇게 구현해보면 어떨까요~?
apiClient는 진희님이 한번 원하시는 방식으로 구현해봅시다 :)
```suggestion
  useEffect(() => {
    async function getProducts() {
      try {
        const products = await apiClient.get('/products');
        setProducts(products);
      } catch (error) {
        openToaster(error.message);
      }
    }

    getProducts();
  }, []);
```

Reply:
네트워크 관련 지식이 짧아 REST API 부분이 나올 때마다 긴장하게 되는 것 같아요 😂 
`async~await` 을 언제 어떻게 써야하는지, error 처리는 어떤 부분에서 처리해줘야 할 지 등
코드를 반영하면서 복습해보게 됐습니다.

제안해주신 코드를 참조해 apiClient 관련 util과 constants를 분리해봤습니다!
`try~catch` 구문은 유틸 함수 내에서 재활용 해 처리할 수 있도록 해봤어요 🙂 - [fc7a9b4](https://github.com/next-step/react-shopping-cart/pull/29/commits/fc7a9b4a4a9eaf7f8c990db310e95cee6066fef1)